README.yaml

```diff
@@ -55,33 +69,55 @@ references:
     description: "The required_version setting can be used to constrain which versions of the Terraform CLI can be used with your configuration"
     url: "https://www.terraform.io/docs/configuration/terraform.html#specifying-a-required-terraform-version"
 
-# Short description of this project
-description: |-
-  This is `terraform-example-module` project provides all the scaffolding for a typical well-built Cloud Posse module. It's a template repository you can
-  use when creating new repositories.
 
 # Introduction to the project
 #introduction: |-
 #  This is an introduction.
 
 # How to use this module. Should be an easy example to copy and paste.
-usage: |-
+uusage: |-
   For a complete example, see [examples/complete](examples/complete).
-  
   For automated tests of the complete example using [bats](https://github.com/bats-core/bats-core) and [Terratest](https://github.com/gruntwork-io/terratest)
   (which tests and deploys the example on AWS), see [test](test).
-
   ```hcl
-  module "example" {
-    source = "https://github.com/cloudposse/terraform-example-module.git?ref=master"
-    example = "Hello world!"
-  }
-  ```
-
+      module "registration" {
+          source = "./registration"
+          enabled = true
+          name    = "clienta-org"
+          namespace = ""
+          subnets = ["subnet-35771e1b", "subnet-801a78e7"]
+          vpc_id = "vpc-8514abff"
+          databases = {
+              "airflow" = {
+                  name    = "clienta-org-airflow"
+                  cluster_size              = 2,
+                  create_random_db_password = true,
+                  instance_type             = "db.t2.small",
+                  security_groups           = [],
+                  retention_period          = 5,
+                  backup_window             = "07:00-09:00",
+                  admin_user                = "admin"
+              },
+              "bioanalyze" = {
+                  name    = "clienta-org-bioanalyze"
+                  cluster_size              = 2,
+                  create_random_db_password = true,
+                  instance_type             = "db.t2.small",
+                  security_groups           = [],
+                  retention_period          = 5,
+                  backup_window             = "07:00-09:00",
+                  admin_user                = "admin"
+              }
+          } 
+          enable_ssl = true
+          install_ingress = true
+          helm_release_values_service_type = "LoadBalancer"
+          create_route53_record = true
+      }
 # Example usage
 examples: |-
   Here is an example of using this module:
-  - [`examples/complete`](https://github.com/cloudposse/terraform-example-module/) - complete example of using this module
+  - [`examples/complete`](https://github.com/dabble-of-devops-biodeploy/terraform-aws-eks-jupyterhub/) - complete example of using this module
```

Comment:
All instances of biodeploy should be bioanalyze. I changed the name several times.

```
[`examples/complete`](https://github.com/dabble-of-devops-bioanalyze/terraform-aws-bioanalyze-onboarding) - complete example of using this module
```